refactor: reorganize directories and improve code readability and sep…

PiewDev opened the pull request:
…aration of concerns

- Reorganized project directories for better structure.
- Enhanced code readability.
- Improved separation of responsibilities across the codebase.

PiewDev committed:
refactor: reorganize directories and improve code readability and separation of concerns

- Reorganized project directories for better structure.
- Enhanced code readability.
- Improved separation of responsibilities across the codebase.

PiewDev committed:
fix: correct route name casing and remove unnecessary config from package.json

- Changed route name to lowercase.
- Removed unnecessary configuration from package.json.

HijoDeHonor commented on src/prompt/messages.js:
Can we change the use of "props[QUESTIONS.....]" for constants declared at the beginning?

HijoDeHonor reviewed (commented):
We should resolve whether or not the change in the use of props or constants is convenient.

PiewDev committed:
refactor: improve readability based on PR suggestion

- Refactored messages for better clarity and readability as suggested in the PR.
